packages/constants/src/index.ts

```diff
@@ -88,3 +85,26 @@ export const testChainInfo: ChainInfo = {
   chainId: "xion-local-testnet-1",
   chainName: "Xion Testnet Local",
 };
+
+// If mainnet chain-id/network changes be sure to update here.
+const DASHBOARD_URLS = {
+  "xion-mainnet-1": "dashboard.burnt.com",
+  "xion-testnet-1": "testnet-dashboard.burnt.com",
```

Comment:
```suggestion
  "xion-testnet-1": "testnet.dashboard.burnt.com",
```

Reply:
I can make this change but I was under the impression that we wanted to stay away from using too many "." and thus the reason for the hyphenated sub-domain